test/__tests__/data/tiktok_ads_input.json

```diff
@@ -2264,7 +2264,7 @@
         "eventId": "1616318632825_357",
         "products": [
           {
-            "product_id": "123",
+            "product_id": 123,
```

Comment:
Is it fine to also convert other product related parameters like content type or category to string too? Does tiktok work fine with numberical values/ structures like object in its place?

Reply:
same here